src/testUtils/stubs.ts

```diff
@@ -2,3 +2,9 @@ export const ORG_NAME = 'example.com';
 export const NON_ASCII_ORG_NAME = 'はじめよう.みんな';
 export const AWALA_ENDPOINT = 'awala-example.com';
 export const NON_ASCII_AWALA_ENDPOINT = 'はじめよう.みんな';
+export const MEMBER_NAME = 'TEST_MEMBER_NAME';
+export const NON_ASCII_MEMBER_NAME = 'テスト ユーザー';
+export const MEMBER_EMAIL = 'membet@example.com';
+
+// To be fixed
+// export const NON_ASCII_MEMBER_EMAIL = 'member@გამარჯობა.ge';
```

Comment:
This can go as we're no longer planning non-ASCII email address support, unless someone asks for it in the future,.